lib/queuePopulator/IngestionProducer.js

```diff
@@ -69,6 +87,27 @@ class IngestionProducer {
         ], err => done(err, this.resLog));
     }
 
+    getRaftLog(raftId, begin, end, targetLeader, done) {
+        const recordResponse = new ListRecordStream(this.log);
```

Comment:
not sure but maybe to be safe, should we add the on 'error' catch for this?
Also, can we rename this variable to use "stream" instead of "response"?